utils/wesnoth-map-diff/src/images.ts

```diff
@@ -17,11 +17,17 @@ const getDictTerrainType2ImagesPath = async () => {
   const dictTerrainType2ImagesPath = rootByTagName(terrain)
     .terrain_type
     .reduce((acc, terrainType) => {
-      if (terrainType.symbol_image === undefined) {
-        return acc
+      if (terrainType.symbol_image) {
+        acc[terrainType.string.value] = terrainType.symbol_image.value
+      }
+      if (!acc[terrainType.string.value] && terrainType.editor_image) {
+        console.log(`No symbol_image defined for the tile "${terrainType.string.value}"; trying editor_image next`)
+        acc[terrainType.string.value] = terrainType.editor_image.value 
+      }
+      if (!acc[terrainType.string.value]) {
+        console.log(`No image defined for the tile "${terrainType.string.value}"; falling back to default`)
+        acc[terrainType.string.value] = 'grid'
       }
```

Comment:
Since we already define a fallback image on lines 52-55, using `blank-hex.png`, we probably don't need it

Reply:
I think it's worth it to be safe

Reply:
...but, I guess I can remove it anyways...

Reply:
ok, removed in [e4c6a36](https://github.com/wesnoth/wesnoth/pull/7499/commits/e4c6a36a176041e93c8bc0ae6e6dafd1a8556e45)